From 4f35359633178d296ed42e29b1d7ae962e450732 Mon Sep 17 00:00:00 2001
Message-ID: <4f35359633178d296ed42e29b1d7ae962e450732.1705743627.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 23:18:18 +0000
Subject: [PATCH 13/38] gnu: go-github-com-marten-seemann-qtls: Move to (gnu
 packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-marten-seemann-qtls): Move from
here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: I036e80323ac61d91743e2c42f878ecad06e6c9b9
---
 gnu/packages/golang-crypto.scm | 28 +++++++++++++++++++++++++++-
 gnu/packages/golang.scm        | 26 --------------------------
 2 files changed, 27 insertions(+), 27 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 7f25cd12a2..36475ce9d4 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -1,6 +1,6 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2018 Pierre Neidhardt <mail@ambrevar.xyz>
-;;; Copyright © 2020 Leo Famulari <leo@famulari.name>
+;;; Copyright © 2019, 2020 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2022 Nicolas Graves <ngraves@ngraves.fr>
 ;;; Copyright © 2023 Clément Lassieur <clement@lassieur.org>
@@ -176,6 +176,32 @@ (define-public go-github-com-marten-seemann-chacha20
 library's internal ChaCha20 package.")
     (license license:bsd-3)))
 
+(define-public go-github-com-marten-seemann-qtls
+  (package
+    (name "go-github-com-marten-seemann-qtls")
+    (version "0.4.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/marten-seemann/qtls")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0dz60y98nm7l70hamq0v2vrs2dspyr5yqhnrds2dfh7hchxvq76j"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/marten-seemann/qtls"
+       ;; The test suite requires networking.
+       #:tests? #f))
+    (propagated-inputs
+     (list go-golang-org-x-crypto))
+    (home-page "https://github.com/marten-seemann/qtls")
+    (synopsis "TLS 1.3 with QUIC in Go")
+    (description "This package provides @code{qtls}, a QUIC-capable variant of
+the Go standard library's TLS 1.3 implementation.")
+    (license license:bsd-3)))
+
 (define-public go-github-com-multiformats-go-multihash
   (let ((commit "97cdb562a04c6ef66d8ed40cd62f8fbcddd396d6")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 499bf6c8a9..7da82299ad 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -7064,32 +7064,6 @@ (define-public go-github-com-burntsushi-locker
 mutex, simply ignore the RLock/RUnlock functions.")
       (license license:unlicense))))
 
-(define-public go-github-com-marten-seemann-qtls
-  (package
-    (name "go-github-com-marten-seemann-qtls")
-    (version "0.4.1")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                     (url "https://github.com/marten-seemann/qtls")
-                     (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "0dz60y98nm7l70hamq0v2vrs2dspyr5yqhnrds2dfh7hchxvq76j"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/marten-seemann/qtls"
-       ;; The test suite requires networking.
-       #:tests? #f))
-    (propagated-inputs
-     (list go-golang-org-x-crypto))
-    (synopsis "TLS 1.3 with QUIC in Go")
-    (description "This package provides @code{qtls}, a QUIC-capable variant of
-the Go standard library's TLS 1.3 implementation.")
-    (home-page "https://github.com/marten-seemann/qtls")
-    (license license:bsd-3)))
-
 (define-public go-github-com-cheekybits-genny
   (package
     (name "go-github-com-cheekybits-genny")
-- 
2.41.0

